experiment/sync-docker.sh

```diff
@@ -6,6 +6,10 @@ manifest="${PARAM_manifest:-conf/contperf/manifest.zip}"
 inventory="${PARAM_inventory:-conf/contperf/inventory.ini}"
 private_key="${PARAM_private_key:-conf/contperf/id_rsa_perf}"
 
+test_sync_repositories_count="${PARAM_test_sync_repositories_count:-8}"
+test_sync_repositories_url_template="${PARAM_test_sync_repositories_url_template:-https://registry-1.docker.io*}"
+test_sync_repositories_max_sync_secs="${PARAM_test_sync_repositories_max_sync_secs:-600}"
+
```

Comment:
These should be called `test_sync_docker_...` I think. Also no `*` is needed in URL.

Reply:
At least that is how they are called in the playbook I think.

Reply:
okay, removed.

Reply:
I can still see `test_sync_repositories_` instead of `test_sync_docker_` in the commit

Reply:
Yes, you are right. Sorry, I missed it. Now, it changed.